blackjax/mcmc/mala.py

```diff
@@ -0,0 +1,63 @@
+"""Public API for Metropolis Adjusted Langevin kernels."""
+from typing import Callable, NamedTuple, Tuple
+
+import jax
+import jax.numpy as jnp
+
+from blackjax.mcmc.diffusion import overdamped_langevin
+from blackjax.types import PRNGKey, PyTree
+
+__all__ = ["MALAState", "MALAInfo", "init", "kernel"]
+
+
+class MALAState(NamedTuple):
```

Comment:
Nice! It makes sense to keep this simple for now rather than worry too much about accomodation SVRG later on.